lib/ecto/schema.ex

```diff
@@ -1388,13 +1388,11 @@ defmodule Ecto.Schema do
   ## Callbacks
 
   @doc false
-  # TODO: Use utc_now instead.
   def __timestamps__(:naive_datetime, :seconds) do
-    NaiveDateTime.from_erl!(:erlang.universaltime, 0)
+    %{NaiveDateTime.utc_now() | microsecond: {0, 6}}
```

Comment:
setting it as `{0, 0}` would break some pg integration tests: https://gist.github.com/wojtekmach/7e848d91881cb457395c469f6b14850a